app/views/audits/audits/_sidebar.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= form_tag audits_path, method: :get do %>
   <%= render 'audits/common/title' %>
   <%= render 'audits/common/audit_status' %>
+  <%= render 'audits/common/topics' %>
```

Comment:
We've merged https://github.com/alphagov/content-performance-manager/pull/424#issuecomment-349951337 before this, so can you please make sure we put the "Topics" filter in the right place. According to the [Trello card](https://trello.com/c/6h4bHTO1/679-1-filter-area-for-the-my-content-tab), it needs to go after "Primary org" and before "Content type"

Reply:
Addressed in https://github.com/alphagov/content-performance-manager/pull/420/commits/c38fa8e0c2db09e404bff3e0a0dc34663e4465d5